pkg/apiserver/manager/hwameistor/localvolume_controller.go

```diff
@@ -439,7 +440,7 @@ func (lvController *LocalVolumeController) CreateVolumeExpand(volName string, ta
 		return nil, err
 	}
 
-	if quantity.Value() > freeCount {
+	if quantity.Value() > freeCount+requiredCapacityBytes {
```

Comment:
rename variable `quantity` to `targteCapacityBytes` may be easy to understand

Reply:
ok